massgov/mayflower
# React/DP-13800:  Deprecated lifecycle methods

## Description
- (React) [InputTextFuzzy] DP-13800: Adds prop onKeyDownPreventDefault. When set to true, hitting enter without a valid suggestion will trigger event.preventDefault().


## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-13800)

## Steps to Test
1. Pull the branch and ensure all automated tests pass and InputTextFuzzy has no visual changes in functionality.
